src/multichainClient.ts

```diff
@@ -34,8 +58,11 @@ function createBaseClient<T extends RpcApi>(transport: Transport): MultichainApi
       });
     },
     revokeSession: async () => {
-      await transport.request({ method: 'wallet_revokeSession' });
-      await transport.disconnect();
+      try {
+        await transport.request({ method: 'wallet_revokeSession' });
+      } finally {
```

Comment:
why is this change necessary? This might fail?

Reply:
why don't we call `await ensureConnected();` before trying like in the other methods?